app/models/miq_schedule_worker/jobs.rb

```diff
@@ -166,6 +166,10 @@ def database_maintenance_vacuum_timer
     end
   end
 
+  def queue_miq_queue_check_for_timeout
+    queue_work(:class_name => "MiqQueue", :method_name => "check_for_timeout", :zone => nil)
```

Comment:
If we're running this every 15 seconds, do we have to worry about these queueing when an existing one is already in the queue?  I believe the JobProxyDispatcher does a `put_unless_exists`.

Reply:
Looks like the definition of `queue_work` always uses `put_unless_exists`

Reply:
haha, cool, I'm glad we thought of that.

Reply:
I'm sure Keenan loves that